cli/command/formatter/disk_usage.go

```diff
@@ -37,36 +37,56 @@ type DiskUsageContext struct {
 	Volumes     []*types.Volume
 	BuildCache  []*types.BuildCache
 	BuilderSize int64
+
+	trunc bool
 }
 
 func (ctx *DiskUsageContext) startSubsection(format string) (*template.Template, error) {
 	ctx.buffer = bytes.NewBufferString("")
 	ctx.header = ""
-	ctx.Format = Format(format)
+	ctx.Context.Format = Format(format)
 	ctx.preFormat()
 
 	return ctx.parseFormat()
 }
 
 // NewDiskUsageFormat returns a format for rendering an DiskUsageContext
-func NewDiskUsageFormat(source string) Format {
-	switch source {
-	case TableFormatKey:
-		format := defaultDiskUsageTableFormat
-		return Format(format)
-	case RawFormatKey:
-		format := `type: {{.Type}}
+func NewDiskUsageFormat(source string, verbose bool) Format {
+	if verbose {
```

Comment:
I still think a switch here with all the cases would be easier to read.

Reply:
@silvin-lubecki a switch for verbose or switch for source, within the `if verbose` ?

Reply:
What about a switch incorporating all the cases? Like this:
```go
switch{
case ctx.Verbose && source == RawFormatKey:
...
case source == TableFormatKey:
...
case source == RawFormatKey:
...
default:
    return Format(source)
}
```

Reply:
@silvin-lubecki it will be:
```
switch{
case verbose && source == RawFormatKey:
...
case !verbose && source == TableFormatKey:
...
case !verbose && source == RawFormatKey:
...
default:
    return Format(source)
}
```